### DISCUSSION TITLE
OpenWRT куда добавлять стратегии?

### DISCUSSION THREAD

**Discussion by MrStukov** [bol-van/zapret#348] [Q&A] [answered]
"8) Запустите install_easy.sh.
Выберите nfqws или tpws, затем согласитесь на редактирование параметров.
Откроется редактор, куда впишите найденные стратегии." 

Никакой редактор не открывается, у меня есть отобранные стратегии, куда их добавить для их выполнения??

**Comment by numbereleven-a** [comment, marked as answer]
в ридмишке написано, поиск по:
Файл /opt/zapret/config используется различными компонентами системы и содержит основные настройки.

**Comment by bol-van** [comment]
там спрашивают хотите ли редактировать настройки
по умолчанию нет
если нажимаешь yes и редактор не открывается
см сообщения об ошибках
может что-то  терминалом
в openwrt из коробки только vi
для удобства можно установить nano

**Comment by dilikhub** [comment]
opkg update

opkg install nano unzip

wget -O "/tmp/zapret-master.zip" https://github.com/bol-van/zapret/archive/refs/heads/master.zip

cd /tmp

unzip zapret-master.zip

cd /tmp/zapret-master/

./install_easy.sh


nano /opt/zapret/ipset/zapret-hosts-user.txt  
сюда вписать:
googlevideo.com
ggpht.com
ytimg.com
l.google.com
youtube.com
play.google.com
1e100.net
googleusercontent.com
gstatic.com
nhacmp3youtube.com
youtubei.googleapis.com 
youtu.be

nano /opt/zapret/config
сюда вписать:
NFQWS_OPT_DESYNC="--qnum=200 --dpi-desync=disorder2 --dpi-desync-split-pos=1" (у меня лучше это)
либо
NFQWS_OPT_DESYNC="--dpi-desync=fake,split2 --dpi-desync-ttl=4 --dpi-desync-ttl6=2 --dpi-desync-split-pos=1 --wssize 1:6 --dpi-desync-fooling=md5sig"


/etc/init.d/zapret restart
перезапустьть zapret

**Reply by bol-van** [reply]
qnum туда не надо писать. сломаете логику скриптов

**Comment by dilikhub** [comment]
Только у меня на Xiaomi 4a Gigabit edition вай фай 2.4, отвалился через час работы, правда я грузил его с четырех устройств, роутер сильно греется. Нужен более мощный аппарат для таких экспериментов.

**Reply by numbereleven-a** [reply]
> Только у меня на Xiaomi 4a Gigabit edition вай фай 2.4, отвалился через час работы, правда я грузил его с четырех устройств, роутер сильно греется. Нужен более мощный аппарат для таких экспериментов.

на RT-N56U первой ревизии, который чуть хуже этого, всё работало без проблем

**Reply by dilikhub** [reply]
> > Только у меня на Xiaomi 4a Gigabit edition вай фай 2.4, отвалился через час работы, правда я грузил его с четырех устройств, роутер сильно греется. Нужен более мощный аппарат для таких экспериментов.
> 
> на RT-N56U первой ревизии, который чуть хуже этого, всё работало без проблем

Я думаю дело в экономии на охлаждающих элементах, 2 -3 телефона потянет
